_vale/config/vocabularies/Docker/accept.txt

```diff
@@ -153,6 +153,7 @@ kubectl
 kubefwd
 kubelet
 lookup
+label_file
```

Comment:
Adding this to the vocabulary is not the right way to fix this lint error

If this is to disable the sentence case check, you should disable that rule instead.

Reply:
OK, but there are a lot more headings that are flagged for the same thing in this and the other compose file docs. You could disable spellcheck, or better, wrap the options in code spans, to bypass this rule. Adding all of these params to the vocabulary doesn't seem like a good solution